data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -88,6 +99,10 @@ static Event fromMessage(String message) {
                 .build();
     }
 
+    static String getZonedDateTime() {
```

Comment:
I don't believe that we want this.

There could be value in having a `getEventTimestamp()` field on `Event`. But, that would be different. And I believe that it should pull from the Map, not have a duplicated field.